src/Patat/Presentation/Display.hs

```diff
@@ -241,6 +244,15 @@ prettyBlocks :: Theme -> [Pandoc.Block] -> PP.Doc
 prettyBlocks theme = PP.vcat . map (prettyBlock theme) . filter isVisibleBlock
 
 
+--------------------------------------------------------------------------------
+prettyBlocks' :: Theme -> Int -> [Pandoc.Block] -> PP.Doc
+prettyBlocks' theme margin =
+    let
+        spaces = mconcat (replicate margin PP.space)
+        indent = PP.indent (PP.NotTrimmable spaces) (PP.NotTrimmable spaces) in
+    PP.vcat . map indent . map (prettyBlock theme) . filter isVisibleBlock
```

Comment:
Can this be moved to `withWrapSettings`?  I am not completely sure but I think something like `PP.wrapAt (...) . PP.indent (...margin..) (..margin..)` would do the trick, and that way we only need to pass the margin to `withWrapSettings`.

Reply:
Didn't have time to take a look at this part yet, I'll see later